FinderCompass/src/main/java/atomicstryker/findercompass/common/DefaultConfigFilePrinter.java

```diff
@@ -290,8 +292,14 @@ else if (settingList.isEmpty())
 			} else {
 				int[] ids;
 				for(int damage = 0; (ids=OreDictionary.getOreIDs(new ItemStack(pBlock, 1, damage))).length > 0; damage++){

```

Comment:
You are blindly iterating damage values upward ... what if there are gaps? What if all items of any damage match the oredict entry? I would search for the infinite loop here